### ISSUE TITLE
Quest:Vahlarriel's Search (2) missing text

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5607]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Upon speaking to Vahlarriel for quest completion, there is no quest text displayed.

**Expected behaviour**: Tell us what should happen instead
There should be the following quest text:

"Infuriating! This entire fiasco!

The two of them were warned about how dangerous coming here would be. Now they're both missing... or worse.

You said you saw tracks heading to a fort nearby? That may be our best bet.

From what the locals have said, that fortress is infested with members of the Burning Blade--orcs who have dedicated themselves to demon causes. They may not represent the majority of the Horde's beliefs, but they are evil and will slay you on sight if you're caught spying."

**Steps to reproduce the problem**:

1. Start the Vahlarriel's Search quest chain and reach part 2 of the quest
2. Upon retrieving the necklace return to Vahlarriel Demonslayer at Nijel's point to complete quest
3. Note the missing quest text

**Include proofs for this behaviour**
http://wowwiki.wikia.com/wiki/Quest:Vahlarriel%27s_Search_(2)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://wowwiki.wikia.com/wiki/Quest:Vahlarriel%27s_Search_(2)
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![image](https://user-images.githubusercontent.com/32976781/46242264-30ee5980-c3f9-11e8-935e-32f4eeb257d4.png)
